weaviate/Chart.yaml

```diff
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 16.3.0
+version: 16.4.0
```

Comment:
Can you leave this one out? We will change the version ourselves when we need to make a release.

Reply:
got it, I've reverted changes